mvvmfx-validation/pom.xml

```diff
@@ -0,0 +1,71 @@
+<!--
+  Copyright 2013 Alexander Casall, Manuel Mauky
+  
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+    http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>de.saxsys</groupId>
+		<artifactId>mvvmfx-parent</artifactId>
+		<version>1.6.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>mvvmfx-validation</artifactId>
+	<packaging>jar</packaging>
+
+	<name>mvvmFX validation</name>
+	<description>This module contains some validators for JavaFX and mvvmFX</description>
```

Comment:
please update this description according to the review comment above for Readme.md